pkg/pb/scope.proto

```diff
@@ -0,0 +1,19 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+syntax = "proto3";
+
+option go_package = "pb";
+
+package scope;
+
+import "gogo.proto";
+import "encryption_access.proto";
+
+message Scope {
+    string satellite_addr = 1;
```

Comment:
Should we use here both NodeID and address?

Reply:
could be good - i think in general we need to figure out how to not get MITM'd for satellite addresses without node ids. should we add signatures from the standard CA hierarchy to our ca cert chain? basically, satellites need valid certificates signed from a trusted CA?

Reply:
this would require changing our transport client to validate hostnames against trusted CAs in these cases

Reply:
I think that having proper CA shouldn't be a problem, since they are most likely going to expose the satellite console anyways.